integration-tests/steps/change_sets.py

```diff
@@ -196,10 +196,10 @@ def step_impl(context, change_set_name, stack_name):
     )
 
     del response["ResponseMetadata"]
-    for output in context.output:
+    for stack, output in context.output.items():
```

Comment:
will `.items()` have issues in Python2 in this case?

Reply:
It will work, but be slightly less performant than in Python 3 (for Py2 we should really use `iteritems` for that behaviour). Functionally it is compatible, though, and we use `items` a lot now.

Reply:
I suspect that any performance difference here will be completely dwarfed by the network calls to AWS, anyway.

Reply:
Yeah, we use `items` a lot, and I guess I could've thrown away the response for `stack` and used `.values()` instead.